serverlessworkflow/modules/ROOT/nav.adoc

```diff
@@ -29,6 +29,7 @@
 *** xref:tooling/serverless-logic-web-tools/serverless-logic-web-tools-enable-kogito-swf-visualization.adoc[Enabling Kogito Serverless Workflow Visualization in {serverless_logic_web_tools_name}]
 * Service Orchestration
 ** xref:service-orchestration/orchestration-of-openapi-based-services.adoc[Orchestrating the OpenAPI services]
+** xref:service-orchestration/orchestration-of-asyncapi-based-services.adoc[Orchestrating the AsyncAPI services]
```

Comment:
Please also add the card to https://github.com/kiegroup/kogito-docs/blob/main/serverlessworkflow/modules/ROOT/pages/index.adoc